build.sbt

```diff
@@ -9,7 +9,7 @@ import uk.gov.hmrc.versioning.SbtGitVersioning.autoImport.majorVersion
 
 val appName = "birth-registration-matching"
 
-lazy val plugins : Seq[Plugins] = Seq(play.sbt.PlayScala, SbtAutoBuildPlugin, SbtGitVersioning, SbtDistributablesPlugin, SbtArtifactory)
+lazy val plugins : Seq[Plugins] = Seq(play.sbt.PlayScala, SbtGitVersioning, SbtDistributablesPlugin, SbtArtifactory)
```

Comment:
SbtGitVersioning and SbtArtifactory can be removed here (auto-build handles them)

Reply:
will get rid of them